docs/GettingStartedGuide.md

```diff
@@ -1,248 +1,3 @@
-    // for this example, it is assumed that app is server from project root
-    decoderUrl: "harp-gl-decoders.bundle.js"
-    // note, this URL may vary depending on configuration of webpack
-    // for this example, it is assumed that webpack emits bundles to project root
-});
-```
-
-Next, you have to initialize default view settings like camera height over ground and location of
-map center:
-
-```javascript
-// index.js
-import { GeoCoordinates } from "@here/harp-geoutils";
-
-// ...
-mapView.camera.position.set(0, 0, 800);
-mapView.geoCenter = new GeoCoordinates(40.70398928, -74.01319808, 0);
-mapView.resize(mapCanvas.clientWidth, mapCanvas.clientHeight);
-```
-
-### Attach data source
-
-Last step is adding some
-[`VectorDataSource`](https://www.harp.gl/docs/master/doc/classes/harp_vectortile_datasource.vectortiledatasource-1.html)
-to our `MapView` instance:
-
-```javascript
-import {
-    APIFormat,
-    AuthenticationTypeMapboxV4,
-    VectorDataSource
-} from "@here/harp-vectortile-datasource";
-
-const dataSource = new VectorDataSource({
-    baseUrl: "https://vector.hereapi.com/v2/vectortiles/base/mc",
-    authenticationCode: "YOUR-APIKEY"
-});
-mapView.addDataSource(dataSource);
-```
-
-You need to [obtain an apikey](#credentials) to replace `YOUR-APIKEY` and use the service to download vector tiles.
-
-### Enable user interaction with map
-
-What we've achieved so far is basic, static non-interactive. If you want to enable control of map
-like panning, rotating use
-[`MapControls`](https://www.harp.gl/docs/master/doc/classes/_here_harp_map_controls.mapcontrols.html)
-
-Note, this requires additional module: `npm install --save @here/harp-map-controls`.
-
-```javascript
-import { MapControls } from "@here/harp-map-controls";
-MapControls.create(mapView);
-```
-
-## <a name="examples"></a> Examine examples
-
-To begin with, we suggest taking a look at our most basic example, the equivalent of a `hello_world` in
-the [examples package](../@here/harp-examples/README.md)
-
-## <a name="credentials"></a> HERE Credentials
-
-In order to use some of the HERE Services, you would need to register
-and generate credentials.
-
-First, you need to become a [HERE Developer](https://developer.here.com/).
-
-Afterwards, please read the [Authentication and Authorization Guide](https://developer.here.com/documentation/authentication/dev_guide/index.html).
-
-For [Map Tile API](https://developer.here.com/documentation/map-tile/dev_guide/topics/quick-start-map-tile.html), which is needed for the webtile examples, you need to generate an `apikey`.
-
-For [Vector Tiles](https://developer.here.com/documentation/vector-tiles-api/dev_guide/index.html), you need to generate an `apikey`.
-
-These credentials need to be passed to the Service in order to retrieve tiles, please see the
-examples to check how it is done.
+Please see:https://developer.here.com/tutorials/harpgl/ for further information on how to get started with harp.gl.
```

Comment:
please add a space after the colon to improve readability

Reply:
@dbacarel , done